docker/README.md

```diff
@@ -31,6 +31,10 @@ Initial setup
     If all goes according to plan you should be able to log into CommCare: http://localhost:8000 using
     the login details above.
     
+    You may encounter an authorization error, such as `restkit.errors.Unauthorized: {"error":"unauthorized","reason":"You are not a server admin."}`.  If so, you may need to alter your docker/localsettings.py COUCH_DATABASES setup by adding the following:
+	* 'COUCH_USERNAME': 'commcarehq',
+        * 'COUCH_PASSWORD': 'commcarehq',
```

Comment:
Is this comment still needed since you have made the change to [docker/localsettings.py](https://github.com/dimagi/commcare-hq/pull/18003/files#diff-61fb5d4bf7e411768579934d90561e5f)?